assets/src/blocks/amp-story/edit.js

```diff
@@ -42,13 +43,8 @@ const TEMPLATE = [
 ];
 
 class EditPage extends Component {
-	constructor( props ) {
-		// Call parent constructor.
-		super( props );
-
-		if ( ! props.attributes.anchor ) {
-			this.props.setAttributes( { anchor: uuid() } );
```

Comment:
This removal seems to be causing validation errors due to the page not getting an `id`.

Reply:
D'oh, wrong file 🤦‍♂️